client/my-sites/plans-features-main/index.tsx

```diff
@@ -700,6 +700,9 @@ const PlansFeaturesMain = ( {
 							toggleShowPlansComparisonGrid={ toggleShowPlansComparisonGrid }
 							planTypeSelectorProps={ planTypeSelectorProps }
 							ref={ plansComparisonGridRef }
+							onStorageAddOnClick={ () => {
+								recordTracksEvent( 'calypso_plans_page_storage_add_on_click' );
```

Comment:
**Note:**

I gave some thought to other, relevant information we wanted to track with this event. Given, however, that we're limiting the feature to the signup onboarding flow, I think simply tracking the event is enough. Definitely open to suggestions though